app/models/applicant_fields.rb

```diff
@@ -0,0 +1,34 @@
+class ApplicantFields
+  AGE = [
```

Comment:
Does not really make a difference in usage, but how about starting to use `%w(` syntax here and then `.freeze` these arrays? I think the [% literal syntax](https://docs.ruby-lang.org/en/2.4.0/syntax/literals_rdoc.html#label-Percent+Strings) could be a bit more readable for these lists.